contracts/FountainV1.sol

```diff
@@ -93,138 +94,94 @@ contract FountainV1 {
+            "No MoneyPool owned by this address"
         );
         require(
-            moneyPools[latestMoneyPoolIds[who]].exists,
-            "No MoneyPool found at this address"
+            moneyPools[latestMoneyPoolIds[owner]].exists,
+            "No MoneyPool owned by this address"
         );
-        return moneyPools[latestMoneyPoolIds[who]].redistributionTracker[by];
+        return moneyPools[latestMoneyPoolIds[owner]].redistributionTracker[by];
     }
 
     constructor(address dai) public {
         DAI = dai;
         moneyPoolCount = 0;
     }
 
-    /// @notice Creates a MoneyPool to be sustained for the sending address.
-    /// @param target The sustainability target for the MoneyPool, in DAI.
-    /// @param duration The duration of the MoneyPool, which starts once this is created.
-    /// @param want The ERC20 token desired, currently only DAI is supported.
-    /// @return success If the creation was successful.
-    function createMoneyPool(
-        uint256 target,
-        uint256 duration,
-        address want
-    ) external returns (bool success) {
-        require(
-            latestMoneyPoolIds[msg.sender] == 0,
-            "Fountain::createMoneyPool: Address already has a MoneyPool, call `update` instead"
-        );
-        require(
-            duration >= 1,
-            "Fountain::createMoneyPool: A MoneyPool must be at least one day long"
-        );
-        require(
-            want == DAI,
-            "Fountain::createMoneyPool: For now, a MoneyPool can only be funded with DAI"
-        );
-
-        moneyPoolCount++;
-        // Must create structs that have mappings using this approach to avoid
-        // the RHS creating a memory-struct that contains a mapping.
-        // See https://ethereum.stackexchange.com/a/72310
-        MoneyPool storage newMoneyPool = moneyPools[moneyPoolCount];
-        newMoneyPool.who = msg.sender;
-        newMoneyPool.sustainabilityTarget = target;
-        newMoneyPool.currentSustainment = 0;
-        newMoneyPool.start = now;
-        newMoneyPool.duration = duration;
-        newMoneyPool.want = want;
-        newMoneyPool.exists = true;
-        newMoneyPool.previousMoneyPoolId = 0;
-
-        latestMoneyPoolIds[msg.sender] = moneyPoolCount;
-
-        emit CreateMoneyPool(
-            moneyPoolCount,
-            msg.sender,
-            target,
-            duration,
-            want
-        );
-
-        return true;
-    }
-
     /// @notice Contribute a specified amount to the sustainability of the specified address's active MoneyPool.
     /// @notice If the amount results in surplus, redistribute the surplus proportionally to sustainers of the MoneyPool.
-    /// @param who Address to sustain.
+    /// @param who The owner of the MoneyPool to sustain.
     /// @param amount Amount of sustainment.
     /// @return success If the sustainment was successful.
     function sustain(address who, uint256 amount)
```

Comment:
Any reason to use `who` here instead of `owner`? Is this due to context, called by sustainer instead of owner ("who" am I sustaining)? I wonder if using `owner` would be better for consistency throughout the full API.

Reply:
I considered this at first. Was curious if you'd also catch it.

Made the change.